### ISSUE TITLE
Syntax Checking not working

### ISSUE THREAD

**Issue by anoadragon453** [godotengine/godot-vscode-plugin#90]
I have intellisense working, am able to run the project through VSCode, and syntax checking is enabled in Settings.

But I can still type `jsajdkdkjasdk` in a `.gd` script and VSCode will not emit any warnings.

![image](https://user-images.githubusercontent.com/1342360/52595177-2206c180-2e45-11e9-8bd4-1c488ca854ca.png)

My platform is Debian Linux using Godot Tools 0.3.7 and VSCode straight from Microsoft's website.

![image](https://user-images.githubusercontent.com/1342360/52595239-4c587f00-2e45-11e9-9869-00615a336bca.png)

I am noticing the following error in VSCode's output tab for "Log (Extension Host):

```
[2019-02-11 21:34:09.463] [exthost] [error] TypeError: Cannot read property 'start' of undefined
	at Object.getSelectedContent (/home/user/.vscode/extensions/geequlim.godot-tools-0.3.7/out/src/gdscript/utils.js:11:45)
	at GDScriptHoverProvider.provideHover (/home/user/.vscode/extensions/geequlim.godot-tools-0.3.7/out/src/gdscript/hoverprovider.js:15:33)
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:539:881
	at t.asPromise (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:50:385)
	at new Promise (<anonymous>)
	at Object.t.asPromise (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:50:353)
	at e.provideHover (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:539:841)
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:556:834
	at e._withAdapter (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:553:292)
	at e.$provideHover (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:556:796)
	at t._doInvokeHandler (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:644:118)
	at t._invokeHandler (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:643:744)
	at t._receiveRequest (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:642:256)
	at t._receiveOneMessage (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:641:155)
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:638:960
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:38:994
	at e.fire (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:44:113)
	at a (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:170:213)
	at Socket.f._socketDataListener (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:170:434)
	at Socket.emit (events.js:182:13)
	at addChunk (_stream_readable.js:279:12)
	at readableAddChunk (_stream_readable.js:264:11)
	at Socket.Readable.push (_stream_readable.js:219:10)
	at Pipe.onread (net.js:636:20)
[2019-02-11 21:34:18.809] [exthost] [error] [geequlim.godot-tools] provider FAILED
[2019-02-11 21:34:18.810] [exthost] [error] TypeError: Cannot read property 'start' of undefined
	at Object.getSelectedContent (/home/user/.vscode/extensions/geequlim.godot-tools-0.3.7/out/src/gdscript/utils.js:11:45)
	at GDScriptHoverProvider.provideHover (/home/user/.vscode/extensions/geequlim.godot-tools-0.3.7/out/src/gdscript/hoverprovider.js:15:33)
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:539:881
	at t.asPromise (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:50:385)
	at new Promise (<anonymous>)
	at Object.t.asPromise (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:50:353)
	at e.provideHover (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:539:841)
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:556:834
	at e._withAdapter (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:553:292)
	at e.$provideHover (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:556:796)
	at t._doInvokeHandler (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:644:118)
	at t._invokeHandler (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:643:744)
	at t._receiveRequest (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:642:256)
	at t._receiveOneMessage (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:641:155)
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:638:960
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:38:994
	at e.fire (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:44:113)
	at a (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:170:213)
	at Socket.f._socketDataListener (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:170:434)
	at Socket.emit (events.js:182:13)
	at addChunk (_stream_readable.js:279:12)
	at readableAddChunk (_stream_readable.js:264:11)
	at Socket.Readable.push (_stream_readable.js:219:10)
	at Pipe.onread (net.js:636:20)
[2019-02-11 21:38:26.850] [exthost] [error] [geequlim.godot-tools] provider FAILED
[2019-02-11 21:38:26.850] [exthost] [error] TypeError: Cannot read property 'variables' of null
	at genWorkspaceTips (/home/user/.vscode/extensions/geequlim.godot-tools-0.3.7/out/src/gdscript/hoverprovider.js:60:67)
	at GDScriptHoverProvider.provideHover (/home/user/.vscode/extensions/geequlim.godot-tools-0.3.7/out/src/gdscript/hoverprovider.js:162:9)
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:539:881
	at t.asPromise (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:50:385)
	at new Promise (<anonymous>)
	at Object.t.asPromise (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:50:353)
	at e.provideHover (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:539:841)
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:556:834
	at e._withAdapter (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:553:292)
	at e.$provideHover (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:556:796)
	at t._doInvokeHandler (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:644:118)
	at t._invokeHandler (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:643:744)
	at t._receiveRequest (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:642:256)
	at t._receiveOneMessage (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:641:155)
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:638:960
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:38:994
	at e.fire (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:44:113)
	at a (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:170:213)
	at Socket.f._socketDataListener (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:170:434)
	at Socket.emit (events.js:182:13)
	at addChunk (_stream_readable.js:279:12)
	at readableAddChunk (_stream_readable.js:264:11)
	at Socket.Readable.push (_stream_readable.js:219:10)
	at Pipe.onread (net.js:636:20)
```

**Comment by TaberIV** [comment]
I have the same issue and getting the same error

**Comment by Geequlim** [comment]
The new version fixed the problem and will be published when godot 3.2 RC release.

**Comment by Geequlim** [comment]
I'm closing this as the new version has fixed this issue.
You can try the new version of the plugin with godot 3.2 from the [release page](https://github.com/godotengine/godot-vscode-plugin/releases)
